src/main/java/com/rarchives/ripme/ui/MainWindow.java

```diff
@@ -1066,6 +1072,17 @@ private void appendLog(final String text, final Color color) {
         logText.setCaretPosition(sd.getLength());
     }
 
+    /**
+     * Write a line to the GUI log and the CLI log
+     *
+     * @param line the string to log
+     * @param color the color of the line for the GUI log
+     */
+    public void dobuleLog(String line, Color color) {
```

Comment:
"dobule" -> "double"

Or better, be more descriptive with something like `displayAndLogError`

Reply:
Done in my last push